ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -360,7 +360,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
         workspace: {
           name: '',
           dataAccessLevel: DataAccessLevel.Registered,
-          namespace: userProfileStore.getValue().profile.freeTierBillingProjectName,
+          namespace: '',
```

Comment:
Should there not still be a namespace for this one? Or is it just never needed?

How did we decide to use empty strings instead of null?

Reply:
It is automatically filled, but I believe required. I can try it as null! /shrug

Reply:
Actually, only name is required! I'll get rid of it.